smoke-tests/apps/SamplingOverrides/src/main/java/com/microsoft/applicationinsights/smoketestapp/ExceptionWithoutSamplingOverridesServlet.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.applicationinsights.smoketestapp;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/trackExceptionWithoutSamplingOverrides")
+public class ExceptionWithoutSamplingOverridesServlet extends HttpServlet {
+
+  protected void doGet(HttpServletRequest request, HttpServletResponse response) {
+    throw new RuntimeException("this is an expected exception without sampling overrides");
+  }
+}
```

Comment:
I don't think you need this servlet, can just hit the same `"/trackException"` url